README.md

```diff
@@ -144,5 +160,22 @@ Using the same front-end application and Rails API from the previous question, h
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
 ```js
-// Your answer goes here...
-```
+$('#ok').click( function () {
+  $.ajax({
+    url: "/posts",
+    type: 'POST',
+    dataType: 'json',
+
+    data:{
```

Comment:
Close, but the strong params means that you'll need to wrap the data inside of a `post:` key.